docs/sources/enterprise/usage-insights.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Usage-insights"
+description = "Usage-insights"
+keywords = ["grafana", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/"]
+type = "docs"
+[menu.docs]
+name = "Usage-insights"
+parent = "enterprise"
+weight = 700
++++
+
+# Usage insights
+
+Usage insights allows you to have a better understanding of how your Grafana instance is used. The collected data are: dashboard views (aggregated and per user), data source errors, and number of data source queries.
```

Comment:
I'd make the list of data collected into a bullet point list to make it more clear.

Reply:
I made an update, changed the sentence a little bit, is it better now?